src/components/Sykmelding/Sykmeldingsperiode.tsx

```diff
@@ -1,61 +1,60 @@
-import { Button, ErrorMessage, TextField } from '@navikt/ds-react';
-import { useFieldArray, useFormContext } from 'react-hook-form';
+import { Button } from '@navikt/ds-react';
+import { Calender } from '@navikt/ds-icons';
+import { useFieldArray, useFormContext, useWatch } from 'react-hook-form';
+
+import SykmeldingSection from '../SykmeldingSection/SykmeldingSection';
+import PeriodeSelect, { Periodetype } from '../FormComponents/Sykmeldingsperiode/PeriodeSelect';
+import GradInput from '../FormComponents/Sykmeldingsperiode/GradInput';
 
 import { SykmeldingFormValues } from './SykmeldingForm';
+import styles from './Sykmeldingsperiode.module.css';
 
 export interface Periode {
-    periode: string;
-    grad: string;
+    sykmeldingstype: string;
+    grad?: number;
 }
 
 function Sykmeldingsperiode(): JSX.Element {
-    const {
-        register,
-        formState: { errors },
-        clearErrors,
-    } = useFormContext<SykmeldingFormValues>();
+    const { control, clearErrors } = useFormContext<SykmeldingFormValues>();
 
     const { fields, append, remove } = useFieldArray({
+        control,
         name: 'periode',
     });
 
+    const watchFieldArray = useWatch({ name: 'periode' });
```

Comment:
får du ikke tilgang til disse via `useFieldArray`, så slipper man en ekstra watch her? Mulig jeg tar feil :smile:

Reply:
det krever ein re-render for å få verdien frå fields